.custom_wordlist.txt

```diff
@@ -166,4 +166,10 @@ jujucharms
 maas
 snapcraft
 snapcraftcontent
-
+MDS
```

Comment:
Can you add the new ones in alphabetical order. Note all of them are in alphabetical order except for proxy stuff